stories/dooboo-ui/Checkbox.stories.tsx

```diff
@@ -57,6 +57,40 @@ function Default(): React.ReactElement {
             },
           ]}
         />
+
+        <Selected>checkbox disabled</Selected>
+
+        <CheckboxGroup
+          boxSize={20}
+          boxColor={'orange'}
+          labelSize={20}
+          labelColor={'#000000'}
+          selectedValue={[
+            {
+              label: 'first',
+              value: 1,
+              selected: true,
+            },
+            {
+              label: 'second',
+              value: 2,
+              selected: false,
+            },
+            {
+              label: 'checked disabled value',
+              value: 3,
+              selected: true,
+              disabled: true,
+            },
+            {
+              label: 'unchecked disabled value',
+              value: 4,
+              selected: false,
+              disabled: true,
+            },
+          ]}
```

Comment:
It's not a necessary modification, but how about putting the data-related part like this at the top?

Reply:
I think this is good for the readability of the code. thanks! 👍